src/Extensions/Compatability/TimedScopes/TimedScopeDefinition.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+//Old namespace used to be compatible with TimedScopeGen
+namespace Microsoft.Omex.Extensions.Compatability.TimedScopes
+{
+	/// <summary>
+	/// Store Timed Scope name and its description
+	/// </summary>
+	[Obsolete(TimedScopeDefinitionExtensions.ObsoleteMessage, TimedScopeDefinitionExtensions.IsObsoleteError)]
+	public class TimedScopeDefinition
+	{
+		/// <summary>
+		/// Name
+		/// </summary>
+		public string Name { get; }
+
+
+		/// <summary>
+		/// Description
+		/// </summary>
+		public string Description { get; }
+
+
+		/// <summary>
+		/// Description
+		/// </summary>
+		public string LinkToOnCallEngineerHandbook { get; }
+
+
+		/// <summary>
+		/// Should the scope be logged only when explicitly demanded
+		/// </summary>
+		public bool OnDemand { get; }
```

Comment:
Based on the description, something like `LogOnlyOnDemand` would be clearer I think.

Applies throughout this class

Reply:
I've moved class to TimedScopes package and deleted fields that are not used